ts/src/hitbtc.ts

```diff
@@ -850,7 +850,7 @@ export default class hitbtc extends Exchange {
             for (let j = 0; j < rawNetworks.length; j++) {
                 const rawNetwork = rawNetworks[j];
                 const networkId = this.safeString2 (rawNetwork, 'protocol', 'network');
-                const network = this.safeNetwork (networkId);
+                const network = this.networkIdToCode (networkId);
```

Comment:
@ttodua  I'm not sure you can just change the behavior without testing, in this case we were returning every network uppercased and that will not happen with `networkIdToCode`....

Reply:
pushed